module/zfs/zfs_ioctl.c

```diff
@@ -2521,6 +2521,26 @@ zfs_prop_set_special(const char *dsname, zprop_source_t source,
 	return (err);
 }
 
+static boolean_t
+zfs_is_namespace_prop(zfs_prop_t prop)
```

Comment:
Out of curiosity, where does "namespace" come from?

Reply:
Dunno.  I copied that part from libzfs, in lib/libzfs/libzfs_dataset.c .